src/ORM/DataList.php

```diff
@@ -1640,9 +1626,6 @@ public function byIDs($ids)
      * Return the first DataObject with the given ID
      *
      * The object returned is not cached, unlike {@link DataObject::get_by_id()}
-     *
-     * @param int $id
-     * @return DataObject|null
```

Comment:
Why is this being removed? Return type should be `T|null`?

Reply:
https://github.com/silverstripe/silverstripe-framework/pull/11108#discussion_r1451866912